e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -244,9 +244,13 @@ test('Example Imagery in Display layout', async ({ page }) => {
     // Click text=Example Imagery
     await page.click('text=Example Imagery');
 
-    // Clear and set Image load delay (milliseconds)
-    await page.click('input[type="number"]', {clickCount: 3});
-    await page.type('input[type="number"]', "20");
+    // Clear and set Image load delay to minimum value
+    // FIXME: This defaults to 5000 ms in execution, but there's a bug
+    // which causes the historical images to not populate for certain delay values.
+    // Update the value to 5000 ms when the bug is fixed.
+    // See: https://github.com/nasa/openmct/issues/5265
```

Comment:
I would maybe add the testannotation for this github issue so that we can track it in the test reporting